serpentTools/tests/test_ResultsReader.py

```diff
@@ -0,0 +1,233 @@
+"""Test the results reader."""
```

Comment:
General comment for these readers. The way the Testers are written, there is no need for the helpers. One specific helper is used to build one specific tester. Since the tests are *largely* the same, just the data compared is different, the actual tests should be put inside a single helper, with different subclasses that test a specific use case. Minimal example
```
class _Helper(unittest.TestCase):
  def test_1(self):
    <do test 1 using instance attributes>
  def test_2(self):
    <do test 2 using instance attributes>
...
class Case1Tester(_Helper):
  @classmethod
  def setupclass(cls):
    <get the reader using the specific settings>
    <set a variety of expected results to be tested against using methods on _Helper>
<repeat for all test cases, e.g. full file, filtered file, file with no universes, file with no time-dependence, etc>
```